.github/workflows/condarise.yaml

```
@@ -40,3 +41,12 @@ jobs:
         run: |
           anaconda -t ${{ secrets.ANACONDA_TOKEN }} \
           upload ${CONDA_PREFIX}/conda-bld/**/*.tar.bz2
+      
+      - name: Redeploy the analysis-server
+        run: |
+          curl --fail \
+            -X POST \
+            -H "Authorization: token ${{ secrets.ANALYSIS_SERVER_GITHUB_TOKEN }}" \
+            -H "Accept: application/vnd.github.v3+json" \
+            https://api.github.com/repos/populationgenomics/analysis-runner/actions/workflows/6364059/dispatches \
+            -d "{\"ref\": \"main\", \"inputs\": {\"hail_version\": \"${{ env.version }}\"}}"
```

```suggestion
            -d "{\"ref\": \"main\", \"inputs\": {\"hail_version\": \"${{ env.version }}\"}}"

```

GH likes empty lines at the end of YAML files, but feel free to ignore this.